packages/charts/theme/lumo/vaadin-chart-styles.js

```diff
@@ -0,0 +1,90 @@
+import '../vaadin-chart-base-theme.js';
+import '@vaadin/vaadin-lumo-styles/color.js';
```

Comment:
Make sure `--lumo-font-family` is defined.

```suggestion
import '@vaadin/vaadin-lumo-styles/color.js';
import '@vaadin/vaadin-lumo-styles/typography.js';
```